### PR TITLE
Add missing 'params' field to pushInstantSpell table

### PR DESCRIPTION
**Pull request by emil92b**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

**Issues addressed:** 
params field was missing

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/luascript.cpp [hunk 1]
```diff
@@ -933,6 +933,7 @@ void LuaScriptInterface::pushInstantSpell(lua_State* L, const InstantSpell& spel
 	setField(L, "mlevel", spell.getMagicLevel());
 	setField(L, "mana", spell.getMana());
 	setField(L, "manapercent", spell.getManaPercent());
+	setField(L, "params", spell.getHasParam());
```
Comment (EPuncker) [inline src/luascript.cpp:936]: I think you need to change line 928 value from 6 to 7 too, since now table will have 7 values

**Review by MillhioreBT — COMMENTED** [review]
I don't see much use for this, I think none, but if you have an example please I want to see
I like to learn from other people who know more ;)

**Review by EPuncker — APPROVED** [review]

File: src/luascript.cpp [hunk 2]
```diff
@@ -925,14 +925,15 @@ void LuaScriptInterface::pushCombatDamage(lua_State* L, const CombatDamage& dama
...
-	lua_createtable(L, 0, 6);
+	lua_createtable(L, 0, 7);
 
 	setField(L, "name", spell.getName());
 	setField(L, "words", spell.getWords());
 	setField(L, "level", spell.getLevel());
 	setField(L, "mlevel", spell.getMagicLevel());
 	setField(L, "mana", spell.getMana());
 	setField(L, "manapercent", spell.getManaPercent());
+	setField(L, "params", spell.getHasParam());
```
Comment (Zbizu) [inline src/luascript.cpp:936]: `hasParam` is a boolean type, not number. I suggest renaming the field to `hasparam` and removing `> 0` from the other commit.

**Review by Zbizu — CHANGES_REQUESTED** [review]

File: src/luascript.cpp (hunk 2 above)
Comment (emil92b) [reply src/luascript.cpp:936]: i don't agree with the rename, unless you do it in spells.xml aswell
but il fix the other commit sure

Comment (Zbizu) [reply src/luascript.cpp:936]: understandable
looks good to me now

**Review by Zbizu — APPROVED** [review]

**Review by EPuncker — APPROVED** [review]

**Pull request merged**